### DISCUSSION TITLE
Welcome to duality Discussions!

### DISCUSSION THREAD

**Discussion by ilexp** [AdamsLair/duality#879] [General]
## Welcome

Hey everyone! 👋 Time to check out this new GitHub discussions feature.

Duality went through a few different stages when it comes to community interaction by now. We started with e-mail between the first few users, switched to a phpbb forum back in the day, then [gradually replaced it](https://github.com/AdamsLair/duality/issues/707) with [the chat](https://chat.duality2d.net/), more or less by accidental preference. And now this!

The chat definitely has its advantages. But it also provides something very different from what you'd typically want for things like "how to use" questions, or more long-term, asynchronous conversation. You can't really link posts, it's not great for archiving information, and invisible to search engines. It does provide a great platform to just hang out though. So why not both? 🙂 

Here's what I think the discussions area could be great for:

- Asking for help in using Duality.
- Sharing ideas, best practices and technical insight.
- Brainstorming ideas before they're ready to become issues / feature proposals.
- Providing general feedback that is not strictly related to development topics or more general in nature.
- Showcasing stuff you built or developed.
- An alternative place to hang out for those who prefer this form of conversation to the chat.

Let's see what this new GitHub discussions feature can do 🎉

**Comment by Rick-van-Dam** [comment]
I wasn't even aware this feature existed in github.

Having a forum like environment in addition to the chat is nice. Even better since is managed by github and 'close' to the code and issues.

**Reply by ilexp** [reply]
Neat, it even seems to have hierarchical discussions.

**Reply by SirePi** [reply]
Oooh, new fancy stuff

**Reply by ChristianGreiner** [reply]
Awesome 👍

**Reply by beljoda** [reply]
Yay. I was missing the old forum a lot lately!
